docs/core/tools/dotnet-new-details.md

```diff
@@ -0,0 +1,69 @@
+---
+title: dotnet new details
+description: The dotnet new details command displays template package metadata.
+ms.date: 06/07/2023
+---
+# dotnet new details
+
+**This article applies to:** ✔️ .NET 8 preview 6
+
+## Name
+
+`dotnet new details` - Displays template package metadata.
+
+## Synopsis
+
+```dotnetcli
+dotnet new details [<PACKAGE_NAME>] [--interactive] [--add-source|--nuget-source <SOURCE>] 
+    [--force] [-d|--diagnostics] [-h|--help]
+```
+
+## Description
+
+The `dotnet new details` command displays the metdata of the template package from the package name provided. For information on a specific version specify the version in with the `--version (check if this is correct)` option. By default, the command searches for the latest available version.
+If the package is installed locally or is found on the official NuGet website, it also displays the templates that the package contains, otherwise it will only display basic metadata.
```

Comment:
```suggestion
If the package is installed locally or is found on the official NuGet website, it also displays the templates that the package contains, otherwise it only displays basic metadata.
```